packages/manager/modules/catalog-price/src/catalog-price.html

```diff
@@ -12,6 +12,12 @@
                         data-translate="{{:: 'order_catalog_price_interval_' + $ctrl.interval}}"
                     ></span>
                 </span>
+                <span data-ng-if="$ctrl.intervalUnit">
+                    &#47;
+                    <span
+                        data-translate="{{:: 'order_catalog_price_interval_' + $ctrl.intervalUnit }}"
+                    ></span>
+                </span>
```

Comment:
It is confusing as to why we need both `interval` and `intervalUnit`. Seems like both result in the same text being rendered... Can't we remove one of them?

Reply:
@ganeshkumar1989 No because `interval` specifies the duration for the given price so let's say I set it to month it will divide the price per 12